### ISSUE TITLE
Hierarchical/Nested State machine inter-exchangeable format

### ISSUE THREAD

**Issue by GeorgeS2019** [imjp94/gd-YAFSM#19]
Great job for YAFSM for godot.

Ideally, Godot YAFSM can import (e.g. json or xml format) FSM file from other Hierarchical/Nested viewers.

Here is an interesting idea to deal with nested FSM

![image](https://user-images.githubusercontent.com/49812372/116561158-5a015e80-a902-11eb-9414-d190f48ea57a.png)

from this [link](https://github.com/miroiu/nodify/issues/6#issuecomment-818968383)

Perhaps the attached view suggest an interesting idea how to display nested FSM in Godot YAFSM. :-)

**Comment by imjp94** [comment]
Yes, I think it would be a nice feature that you can work on your favorite software and import the logic to godot through YAFSM.
But I think this will only happens after v1.0, as I am busy with adding feature now and preparing stable release for v1.0 with better UI/UX.

Can you show some example of FSM editor where it can export into .json/.xml? As I couldn't found doc related to exporting from nodify.

Btw, UI of nodify looks really good, it can be a great source of inspiration for the upcoming UI improvement for YAFSM.
